infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInternalRuntime.java

```diff
@@ -625,6 +691,20 @@ protected void doStartMachine(KubernetesServerResolver serverResolver)
     LOG.debug("Pods creation finished in workspace '{}'", workspaceId);
   }
 
+  private void startTracingContainersStartup(Pod pod) {
+    for (Container container : pod.getSpec().getContainers()) {
+      String machineName = Names.machineName(pod, container);
+
+      machineStartupTraces.put(
+          machineName,
+          tracer
+              .buildSpan("container-create")
```

Comment:
It's not actually container creation but machine starting which includes container creation, machine bootstrapping, server checking.